File: app/src/main/java/com/ducluanxutrieu/quanlynhanvien/Item/Task.java
package com.ducluanxutrieu.quanlynhanvien.Item;

import java.io.Serializable;

public class Task implements Serializable {
    private String taskContent;
    private String taskTitle;

    public Task() {
    }

    public Task(String taskContent, String taskTitle)  {
        this.taskContent = taskContent;
        this.taskTitle = taskTitle;
    }

    public String getTaskContent() {
        return taskContent;
    }

    public String getTaskTitle() {
        return taskTitle;
    }

    public void setTaskContent(String taskContent) {
        this.taskContent = taskContent;
    }

    public void setTaskTitle(String taskTitle) {
        this.taskTitle = taskTitle;
    }
}

File: app/src/main/java/com/ducluanxutrieu/quanlynhanvien/models/Task.kt
package com.ducluanxutrieu.quanlynhanvien.models

import java.io.Serializable

class Task : Serializable {
    var taskContent: String? = null
    var taskTitle: String? = null
    var keyTask: String? = null

    constructor() {}

    constructor(taskTitle: String, taskContent: String) {
        this.taskContent = taskContent
        this.taskTitle = taskTitle
    }
}
